### DISCUSSION TITLE
Can I collapse a retry node?

### DISCUSSION THREAD

**Discussion by David-Giliotti-Wonder** [argoproj/argo-workflows#12676] [Q&A] [answered]
Hello,

I see that there exists a collapse node feature in argo. I was wondering, if for a retry node I could collapse/uncollapse the parent.

For example: Node A at the moment creates a child Node A (0) when retries are enabled. (Or potentially Node A (n) where n is the amount of retries.)

Is there a way for me to collapse Node A's children into Node A? It would greatly reduce how big the workflow looks if I could enable this by default somehow.

Thanks!

**Comment by agilgur5** [comment, marked as answer]
Have you seen the node filters available in the top left of the graph view? Screenshot below:
![Screenshot 2024-02-17 at 1 29 37 PM](https://github.com/argoproj/argo-workflows/assets/4970083/5a18f7a3-0bd0-4b2e-b30e-154b96667529)

There is a filter to hide all "Retry" nodes there, which may suffice for what you're looking for

**Reply by David-Giliotti-Wonder** [reply]
Thank you, this may work for my case but I seem to have a UI bug where the filter popup goes off the screen so I can't see any of these options. Was this fixed in a specific version?

**Reply by agilgur5** [reply]
What version are you on?

There was a simplification in #11901 to remove JS scroll handling as the CSS is already `position: fixed`, but I don't recall the filter popup being reported as buggy / off screen in the past 6 months

**Reply by David-Giliotti-Wonder** [reply]
I am on v3.4.6 (argo-workflow: 0.22.16). Perhaps I shall upgrade. Thank you for your input!
